app/(main)/league/[leagueId]/entry/[entryId]/week/Week.tsx

```diff
@@ -160,15 +161,18 @@ const Week = ({ entry, league, NFLTeams, week }: IWeekProps): JSX.Element => {
       }
       
       setEntryName(currentEntry.name);
-      let entryHistory = currentEntry?.selectedTeams || [];
+      const entryHistory = currentEntry?.selectedTeams || [];
+      
+      let entryHistoryObject: INFLTeam[] = [];
```

Comment:
Looks like this is redundant because getPickHistory is only ran once. Therefore, the default value on line 47 should suffice. There's no need to resave an empty array if it's already the default value. It will cause unnecessary re-renders.